CromaColorPicker.js

```diff
@@ -6,6 +6,7 @@ const colors = [
   '#f50057', '#db0A5b', '#c51162', '#9c27b0', '#673ab7', '#4b77be', '#2196f3', '#03a9f4', '#00bcd4', '#1bbc9b',
   '#009688', '#4caf50', '#8bc34a', '#cddc39', '#ffeb3b', '#ffc107', '#ff9800', '#ff5722', '#f44336', '#e00032'
 ];
+const fixedColors = ['#000000', '#ff0000', '#ffa500','#ffff00', '#00ff00', '#00ffff', '#0000ff', '#ff00ff','#00ffa5', '#ffffff']
```

Comment:
I suggest changing to:

```suggestion
const fixedColors = ['#000000', '#ff0000', '#ffa500','#ffff00', '#00ff00', '#00ffff', '#0000ff', '#ff00ff','#00ffa5', '#ffffff'].sort();
```

For a more logical sequence of colors